dali/kernels/imgproc/color_manipulation/brightness_contrast_test.cc

```diff
@@ -79,15 +79,23 @@ class BrightnessContrastCpuTest : public ::testing::Test {
 using TestTypes = std::tuple<uint8_t, int8_t, uint16_t, int16_t, int32_t, float>;
 INPUT_OUTPUT_TYPED_TEST_SUITE(BrightnessContrastCpuTest, TestTypes);
 
+namespace {
+
+template <class GtestTypeParam>
+using TheKernel = BrightnessContrastCpu<typename GtestTypeParam::Out, typename GtestTypeParam::In>;
```

Comment:
For me something more verbose like `BrightnessContrastKernel` would make the code below easier to read, but that is just personal preference, I guess.

Reply:
+1

Reply:
Done